diff --git a/html/details.php b/html/details.php
--- a/html/details.php
+++ b/html/details.php
@@ -1170 +1170 @@ function DisplayDetails() {
-				$packets_sum[$_type] += $statinfo['packets' . $_type];
+				$packets_sum[$_type] += intval($statinfo['packets' . $_type]);
@@ -1192 +1192 @@ function DisplayDetails() {
-				$bits_sum[$_type] += $statinfo['traffic' . $_type];
+				$bits_sum[$_type] += intval($statinfo['traffic' . $_type]);
diff --git a/html/nfsenutil.php b/html/nfsenutil.php
--- a/html/nfsenutil.php
+++ b/html/nfsenutil.php
@@ -353,0 +354,2 @@ function ScaleBytes ( $value, $rateval, $bs ) {
+	$value = floatval($value);
+
